runtime/doc/nvim_clipboard.txt

```diff
@@ -56,4 +56,11 @@ Nvim's X11 clipboard providers only utilize the `PRIMARY` and `CLIPBOARD`
 selections, used for the '*' and '+' registers, respectively.
 
 ==============================================================================
+						   *g:loaded_clipboard_provider*
+
+To disable the clipboard feature, set |g:loaded_clipboard_provider| to 1:
+>
+    let g:loaded_clipboard_provider = 1
+<
```

Comment:
Why do we need this? User can already unset `&clipboard`. We should not be in the business of fixing plugin bugs.